### PR TITLE
fix email images etc

### PR DESCRIPTION
**Pull request by rolandpo**
fix email not showing images in analytics email, other styling issues and email settings

issue #269

requires updating variable `DEFAULT_FROM_EMAIL=Origin Dollar <no-reply-ousd-analytics@originprotocol.com>` and adding new variable `REPLY_TO_EMAIL=ousd@originprotocol.com`

### CODE REVIEW

File: eagleproject/core/templates/analytics_report_email.html [hunk 1]
```diff
@@ -18,9 +17,9 @@ <h4 style="margin-top:16px;font-size: 14px;line-height: 16px;">
         UTC.
     </h4>
 
-    <h2 class="d-flex flex-row gap-2" style="margin-top: 48px;font-size: 20px;line-height: 23px;font-weight:bold;">
-      <img src="{{STATIC_ROOT}} /static/core/img/ousd.png" alt="ousd" style="width: 32px;height: 32px;" />
```
Comment (shahthepro) [inline eagleproject/core/templates/analytics_report_email.html:22]: I'm wondering if it wasn't working because of the whitespace in between

**Review by shahthepro — APPROVED** [review]

File: eagleproject/core/migrations/0048_day_alter_analyticsreport_created_at_and_more.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.1.1 on 2023-02-12 12:33
```
Comment (shahthepro) [inline eagleproject/core/migrations/0048_day_alter_analyticsreport_created_at_and_more.py:1]: @rolandpo Can you delete this file from this PR? I have noticed that the changes this migration makes is already on prod DB. So committing them makes the prod deployment to fail

Comment (rolandpo) [reply eagleproject/core/migrations/0048_day_alter_analyticsreport_created_at_and_more.py]: yep, deleted

Comment (shahthepro) [reply eagleproject/core/migrations/0048_day_alter_analyticsreport_created_at_and_more.py]: Thank you

**Pull request merged**